config.json

```
@@ -18,16 +18,16 @@
   },
   "files": {
     "solution": [
-      "path/to/%{kebab_slug}.ext"
+      "%{kebab_slug}/%{snake_slug}.rb"
```

what is the reference for what the path is relative to? I saw @angelikatyborska used the specific exercises' root dir as the implicit root in Elixir and it woule be like this for ruby: 

```suggestion
      "%{snake_slug}.rb"
```

@ErikSchierboom?

---

The example in the documentation is relative to the exercise's directory: https://github.com/exercism/docs/pull/58/files#diff-94a49f096104e9699f32fef95355446ec16df7ca900bece658fff59c3f0fd398R61-R64

---

@neenjaw Yeah, this came up yesterday. I've since made it explicit in the docs that the paths are relative to the exercise's directory: https://github.com/exercism/docs/commit/49197e6cbf7750ea8f68285c051ded59b910f02e Sorry about the confusion